### PR TITLE
Merge remote-tracking branch 'origin/releases' into mc/merge/0.88_e599af1e

### PR DESCRIPTION
**Pull request by beeender**

### CODE REVIEW

File: version.txt [hunk 1]
```diff
@@ -1 +1 @@
-0.89.0-SNAPSHOT
+0.88.2-SNAPSHOT
```
Comment (zaki50) [inline version.txt]: do not merge this change

**Comment by zaki50** [comment]
I think we can simply reject this merge.

Is it OK to close this? @realm/java

**Comment by cmelchior** [comment]
Yes, I think all the changes have already been merged to master?

**Comment by zaki50** [comment]
not yet merged, but already included in other PRs.

**Pull request closed without merging**